library/src/interactor_impl.cxx

```diff
@@ -1608,9 +1634,11 @@ interactor& interactor_impl::initBindings()
   this->addBinding({mod_t::ANY, "1"}, "set_camera front", "Camera", std::bind(docStr, "Front View camera"));
   this->addBinding({mod_t::ANY, "2"}, "elevation_camera -90", "Camera", std::bind(docStr, "Rotate camera down"));
   this->addBinding({mod_t::ANY, "3"}, "set_camera right", "Camera", std::bind(docStr, "Right View camera"));
-  this->addBinding({mod_t::ANY, "4"}, "roll_camera -90", "Camera", std::bind(docStr, "Rotate camera right"));
+  this->addBinding({mod_t::NONE, "4"}, "roll_camera -90", "Camera", std::bind(docStr, "Rotate camera right"));
+  this->addBinding({mod_t::CTRL, "4"}, "rotate_up x 90", "Scene", std::bind(docStr, "Rotate scene up +90 around X"));
```

Comment:
I dont think this is correct, because the axis to rotate around depends on current state ?